addons/website/static/src/components/views/page_kanban.xml

```diff
@@ -1,19 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 
-<t t-name="website.PageKanbanRenderer" t-inherit="web.KanbanRenderer" owl="1">
```

Comment:
@qsm-odoo What is the best approach regarding stable here?
I guess we can't just remove this template like that.
Should I keep it and remove the `PageKanbanRenderer.template = 'website.PageKanbanRenderer';` line (removed in this PR above) so the template still exists in case of extend/xpath but it's not used?

There is a bunch of code not needed anymore that I will delete in master and that I marked as `TODO remove` here, but for this one, I am not sure?

Reply:
Nice, I went with the empty `t-if`, seems very fine to me :+1: 
Should be good to be reviewed or delegated